charts/osm/Chart.yaml

```diff
@@ -14,8 +14,8 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.4.2
+version: 0.5.0-rc.1
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
-appVersion: v0.4.2
+appVersion: v0.5.0-rc.1
```

Comment:
This will change to `v0.5.0` once we have cut a release. You might want to hold off on this PR until that is complete